indigo-sandbox/src/main/scala/example/IndigoSandbox.scala

```diff
@@ -12,7 +13,7 @@ import zio.interop.catz.*
 import scala.scalajs.js.annotation.*
 
 @JSExportTopLevel("TyrianApp")
-object IndigoSandbox extends TyrianApp[Msg, Model]:
+object IndigoSandbox extends SinglePage[Msg, Model]:
```

Comment:
I don't think it's a huge concern since `TyrianApp` is only used here and people bumping Tyrian version would need to change a single line. But it might be worth to keep this type and make it a type alias (or extend from `SinglePage`), what do you think?

Reply:
I'm a bit torn. I'm kinda unafraid of forcing people to keep up in general, but I could put in a type alias with a deprecation warning if I was feeling kind... :smile:

Reply:
I'll do the depreciation warning. Seems the right thing to do.

Reply:
```
trait TyrianApp in package tyrian is deprecated: Please use SinglePage or MultiPage instead of TyrianApp.
```